### PR TITLE
Add diagnostic echo statements to bundler version determination

### PR DESCRIPTION
**Pull request by SeanKilleen**
I believe these statements could be of assistance in diagnosing #383, and could be helpful in general for Netlify's users.

### CODE REVIEW

File: run-build-functions.sh [hunk 1]
```diff
@@ -299,11 +299,15 @@ install_dependencies() {
   if [ -f Gemfile.lock ]
   then
      bundler_version="$(cat Gemfile.lock | grep -C1 '^BUNDLED WITH$' | tail -n1 | sed 's/^[[:blank:]]*//;s/[[:blank:]]*$//' | tr -d \\n)"
+  else
+     echo "Gemfile.lock does not found; not attempting to obtain bundler version from it"
```
Comment (SeanKilleen) [inline run-build-functions.sh]:
```suggestion
     echo "Gemfile.lock not found; not attempting to obtain bundler version from it"
```

**Comment by SeanKilleen** [comment]
Attempting to verify these changes so I can ask for a more formal review from someone, I:

* Pulled the repo
* Built the docker image (`-docker build -t seanlify .`)
* Converted the bash scripts to PowerShell scripts (since I'm running windows)
* Updated the scripts to use my local `seanlify` container image rather than `netlify/build`
* Attempted to use the `test-build.ps1` to start my script.

It looks like in this case the `/user/local/bin/build` executable isn't available.

I'm sure I'm doing something wrong -- putting this out there in case anyone wants to chime in.

**Comment by kitop** [comment]
Closing this PR as the it's against xenial branch and the Xenial image is going to be deprecated: https://answers.netlify.com/t/please-read-end-of-support-for-xenial-build-image-everything-you-need-to-know/68239

Feel free to reopen against focal.

**Comment by SeanKilleen** [comment]
Thanks for the heads up, @kitop. If a PR that has nothing but echo statements and a few helper scripts isn't even responded to for 2 years and then closed, it's probably not important enough to open again.

**Pull request closed without merging**